src/MySQLConnection.ts

```diff
@@ -34,9 +34,17 @@ const DEFAULT_HIGH_WATERMARK: number = 512; // in number of result objects
 const TAG: string = 'MySQLConnection';
 
 const SQL_FORMATTING_OPTIONS: SQLFormatter.FormatOptions = {
-    language: 'mysql',

```

Comment:
Seems odd to be removing this. Can you verify?

Reply:
It was removed because `language` is no longer an accepted key. But looking this into more detail, looks like this change was part of the [v12.0.0](https://github.com/sql-formatter-org/sql-formatter/releases/tag/v12.0.0) breaking change which has an entire new API for using a dialect.

I'll try the new `formatDialect` api and see what happens.